frontend/src/components/AnimatedCounter.tsx

```
@@ -3,19 +3,24 @@
   duration: number
-  end: number
+  end: number | string
+  end: number | string
 }
 
 export default function AnimatedCounter({ end, duration, className }: AnimatedCounterProps) {
-  const [count, setCount] = useState(0)
-  const countRef = useRef(count)
+  const [count, setCount] = useState<string | number>(0)
+  const countRef = useRef(0)
+  const [count, setCount] = useState<string | number>(0)
```

Biome (1.9.4)

[error] Shouldn't redeclare 'count'. Consider to delete it or rename it.

'count' is defined here:


(lint/suspicious/noRedeclare)